apps/mobile/android/app/src/main/assets/messaging/content.js

```
@@ -0,0 +1,101 @@
+  },
+};
+
+window.wrappedJSObject.$onekey = cloneInto(onekeyUtils, window, {
+  cloneFunctions: true,
+});
+
+browser.runtime.onMessage.addListener((data, sender) => {
+  bindWebembedReceiveHandler();
+  if (data.inject) {
+    try {
+      globalThis.eval(data.inject);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)